aktuelt/image_formats.py

```diff
@@ -0,0 +1,15 @@
+# Ref: https://docs.wagtail.org/en/stable/advanced_topics/images/changing_rich_text_representation.html
+from django.utils.html import format_html
+from wagtail.images.formats import Format, register_image_format
+
+
+class InlineNewsImageFormat(Format):
```

Comment:
This formatter is what Wagtail uses to display different image "formats"  that can be selected when inserting an image in editor UI. This example does nothing useful, but outlines how we could use this to define a new format. This could for example be
- A fullscreen image
- An image with or without caption
- An image that includes custom field data
- ...etc